src/services/zapper.ts

```diff
@@ -104,6 +105,29 @@ export class ZapperService extends Service {
     return balances;
   }
 
+  /**
+   * Fetches vault token market data for the Yearn application
+   * @returns list of vault token market data
+   */
+  async tokenMarketData(): Promise<VaultTokenMarketData[]> {
+    if (!isEthereum(this.chainId)) {
+      throw new SdkError(`Only Ethereum is supported for token market data, got ${this.chainId}`);
```

Comment:
Throwing because you should only be able to call `tokenMarketData()` if you're in the ethereum network.